src/import/import.rs

```diff
@@ -375,6 +379,18 @@ impl<Client: ImportClient> ImportSSTJob<Client> {
             let file = self.sst.info.open()?;
             let upload = UploadStream::new(self.sst.meta.clone(), file);
             let store_id = peer.get_store_id();
+            while !self
+                .client
+                .is_space_enough(store_id, self.sst.info.file_size)?
+            {
+                let label = format!("{}", store_id);
+                IMPORT_STORE_SAPCE_NOT_ENOUGH_COUNTER
+                    .with_label_values(&[label.as_str()])
+                    .inc();
+                thread::sleep(Duration::from_millis(
+                    STORE_UNAVAILABLE_WAIT_INTERVAL_MILLIS,
```

Comment:
Stores heartbeat every ten seconds, during that the store available sizee won't change. So it waits too short time here. 
And you'd better print log here.

Reply:
We need a different retry strategy for various activities, so make them configurable is need.

Reply:
addressed

Reply:
no need to print log? I'm afraid we don't know where the process is waiting for.

Reply:
We have metrics to diagnose it.